model/task_lifecycle.go

```diff
@@ -890,14 +890,14 @@ func MarkTaskUndispatched(t *task.Task) error {
 	return nil
 }
 
-func MarkTaskDispatched(t *task.Task, hostId, distroId string) error {
+func MarkTaskDispatched(t *task.Task, host *host.Host) error {
```

Comment:
it's best not to use package names as variable names, because then you can't use that package in this function without renaming the variable anyway.

Reply:
Thanks for the tip! makes sense